yaki_admin/src/ui/components/UserInfoCard.vue

```diff
@@ -23,9 +21,15 @@ const props = defineProps({
 
 const UserToBeRemoved = () => {
   teammateStore.setIdOfTeammateToDelete(props.user.teammateId);
-  modalStore.setTeammateNameToDelete(`${props.user.firstname} ${props.user.lastname}`);
+  modalStore.setTeammateNameToDelete(
+    `${props.user.firstname} ${props.user.lastname}`
```

Comment:
Maybe create a fullname property to make it more clear because used in repetition